src/remoteConnector.ts

```diff
@@ -631,6 +631,7 @@ export default class RemoteConnector extends Disposable {
 			? `You don't have registered any SSH public key for this machine in your Gitpod account.\nAlternatively, copy and use this temporary password until workspace restart: ${maskedPassword}`
 			: `An SSH key is required for passwordless authentication.\nAlternatively, copy and use this password: ${maskedPassword}`;
 		const action = await vscode.window.showWarningMessage(message, { modal: true }, copy, configureSSH, showLogs);
+        this.telemetry.sendRawTelemetryEvent('vscode_desktop_ssh_gateway_modal', { kind: 'gateway', action: action?.title ?? 'Cancelled', ...sshParams, gitpodVersion: gitpodVersion.raw });
```

Comment:
`cancelled` and `configureSSH` options telemetry info is already send in line 716